SwiftKitCore/src/main/java/org/swift/swiftkit/core/ConfinedSwiftMemorySession.java

```diff
@@ -16,28 +16,26 @@
 
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
 import java.util.concurrent.atomic.AtomicInteger;
 
 public class ConfinedSwiftMemorySession implements ClosableSwiftArena {
```

Comment:
Let's open an issue to rename this type since it is not really following the existing pattern in the JDK where Confined was about confining to a specific thread.